src/command_modules/azure-cli-natgateway/azure/cli/command_modules/natgateway/_params.py

```diff
@@ -16,4 +18,5 @@ def load_arguments(self, _):
         c.argument('public_ip_addresses', nargs='+', help='Space-separated list of public IP addresses (names or IDs).', validator=validate_public_ip_addresses)
         c.argument('public_ip_prefixes', nargs='+', help='Space-separated list of public IP prefixes (names or IDs).', validator=validate_public_ip_prefixes)
         c.argument('idle_timeout', help='Idle timeout in minutes.')
+        c.argument('zone', nargs='+', help='Zone of the Nat Gateway')
```

Comment:
Why are you not just using `zone_type`?  If it accepts multiple, then import `zones_type` instead.